src/pages/Admin/Venue/VenueAdminPage.tsx

```diff
@@ -48,13 +58,29 @@ export const VenueAdminPage: React.FC = () => {
     );
   }
 
-  const isIframeVenue = IFRAME_TEMPLATES.includes(venue.template);
```

Comment:
Why was this removed?

Reply:
same as above https://www.figma.com/file/oX3Px4vYjjUkOZYSqcCgWI/Sparkle-Banner-Announcements?node-id=3%3A6